go-update/src/main/java/org/reactome/release/goupdate/GoTermsUpdater.java

```diff
@@ -193,22 +192,53 @@ else if (goTermsFromFile.get(goID).containsKey(GoUpdateConstants.PENDING_OBSOLET
 			}
 			else if (goTermsFromFile.get(goID).containsKey(GoUpdateConstants.IS_OBSOLETE) && goTermsFromFile.get(goID).get(GoUpdateConstants.IS_OBSOLETE).equals(true))
 			{
-				// If we have this in our database, it must be reported!
+				// If we have this in our database, it must be reported as obsolete!
 				if (goInstances!=null)
 				{
 					obsoleteCount++;
-					String replacementValue = goTermsFromFile.get(goID).get(GoUpdateConstants.REPLACED_BY) != null ? " Replacement Accesion: " + goTermsFromFile.get(goID).get(GoUpdateConstants.REPLACED_BY) : " No replacement suggested, GO term will NOT be deleted.";
-					obsoleteAccessionLogger.warn("GO:{} ({}) marked as OBSOLETE!{}",goID, goInstances.toString(), replacementValue);
+					StringBuilder attemptToDeleteObsoleteMessage = new StringBuilder();
+					Map<GKSchemaAttribute, Integer> referrersCount = new HashMap<>();
 					// Only add instance(s) to deletion list if they have a valid replacement.
 					if (goTermsFromFile.get(goID).get(GoUpdateConstants.REPLACED_BY) != null)
 					{
 						for (GKInstance inst : goInstances)
 						{
 							instancesForDeletion.add(inst);
 						}
```

Comment:
Could the for loop be done as: `instancesForDeletion.addAll(goInstances)`?

Reply:
I think that loop used to do more than just adding to another list.